src/lib.rs

```diff
@@ -85,6 +82,11 @@ pub fn run(path: &PathBuf) -> Vec<Message> {
                 );
 
                 if data != Value::None {
+                    if fd.definition_number == 253 {
```

Comment:
I'm a little uncomfortable about hardcoding `253` here, I'd prefer it if we could somehow use the SDK definition to determine if we're dealing with a timestamp value. Do you think this could be moved down a bit and into the 

`FieldType::DateTime => {`

section?

Reply:
No: some messages can have several fields of type datetime, and some others can have no timestamp.
I started to add the field name in the DataField struct, but it's not yet ready to be merged.
I will reopen the PR when it will be finished